core/cmd/remote_client.go

```diff
@@ -182,6 +182,24 @@ func (cli *Client) RemoteLogin(c *clipkg.Context) error {
 	return nil
 }
 
+// Logout removes local and remote session.
+func (cli *Client) Logout(c *clipkg.Context) error {
```

Comment:
I think you will need a named return here for the defer to work properly.
```suggestion
func (cli *Client) Logout(c *clipkg.Context) (err error) {
```

Reply:
Ah dang, great catch. Updated